### PR TITLE
docs(changelog): Add 3.0.2 changelog; fix 2 minor typos

### PR DESCRIPTION
**Pull request by mrienstra**
As per https://github.com/yarnpkg/berry/compare/@yarnpkg/cli/3.0.1...@yarnpkg/cli/3.0.2

Same same:
@yarnpkg/pnp: https://github.com/yarnpkg/berry/compare/@yarnpkg/pnp/3.0.1...@yarnpkg/pnp/3.0.2
@yarnpkg/plugin-pnp: https://github.com/yarnpkg/berry/compare/@yarnpkg/plugin-pnp/3.0.1...@yarnpkg/plugin-pnp/3.0.2

### CODE REVIEW

**Comment by merceyz** [comment]
The TypeScript patch was also backported to 2.4.3 if could add a note for that as well

File: CHANGELOG.md [hunk 1]
```diff
@@ -131,6 +136,23 @@ Yarn now accepts sponsorships! Please give a look at our [OpenCollective](https:
...
 - Reporting for HTTP errors has been improved, which should help you investigate registry issues.
 
+## 2.4.3
+
+```
+yarn set version 2.4.3
+```
+
+- Updated TypeScript patch to cover TypeScript 4.4.
+- Removed `useUnknownInCatchVariables` from `tsconfig.json`.
```
Comment (merceyz) [inline CHANGELOG.md:146]:
```suggestion
```

Internal change, doesn't affect users

**Comment by mrienstra** [comment]
> The TypeScript patch was also backported to 2.4.3 if could add a note for that as well that would be great

Good call, I added 2.4.3 & 2.4.2.

Tempted to add something like:

```md
[Latest v3: 3.0.2](#302)

[Latest v2: 2.4.2](#242)
```
Screen Shot 2021-09-11 at 2 52 54 PM

... Though I don't love the copy or the formatting, just thinking about how to make it a little easier for people still on v2 to jump down quickly. Maybe it would be better to add section headers for v3 & v2, and link to those from something like a table of contents. New sections headers could be `##` (h2) and version headers could be dropped down to `###` (h3).

... But probably better to save that for a separate PR. 😁

File: CHANGELOG.md (hunk 1 above)
Comment (mrienstra) [reply CHANGELOG.md]: Good call

**Comment by merceyz** [comment]
GitHub already has a table of contents for markdown files that you can just type `2` into and you'll see all entries for v2
image

**Comment by mrienstra** [comment]
> GitHub already has a table of contents for markdown files

🤯

**Comment by arcanis** [comment]
Thanks!

**Pull request merged**